test/abi.inputParser.js

```diff
@@ -294,13 +244,13 @@ describe('abi', function() {
 
         });
 
-        it('should parse input string', function () {
+        it('should parse input fixed bytes type', function () {
 
             // given
             var d = clone(description);
 
             d[0].inputs = [
-                { type: "string" }
```

Comment:
This seems not correct. `string` is a dynamic type while `bytes32` is a fixed type. I think either leaving the string as it is or changing it to `bytes` would be the right way.